File: VB/Dashboard_UnderlyingDataWeb/SalesDashboard.vb
﻿Imports System
Imports Dashboard_UnderlyingDataWeb.nwindDataSetTableAdapters

Namespace Dashboard_UnderlyingDataWeb
    Partial Public Class SalesDashboard
        Inherits DevExpress.DashboardCommon.Dashboard

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub SalesDashboard_DashboardLoading(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.DashboardLoading
            Dim categories As New SalesPersonTableAdapter()
            categories.Fill(Me.nwindDataSet1.SalesPerson)
        End Sub
    End Class
End Namespace

File: CS/Dashboard_UnderlyingDataWeb/SalesDashboard.cs
using System;
using Dashboard_UnderlyingDataWeb.nwindDataSetTableAdapters;

namespace Dashboard_UnderlyingDataWeb {
    public partial class SalesDashboard : DevExpress.DashboardCommon.Dashboard {
        public SalesDashboard() {
            InitializeComponent();
        }

        private void SalesDashboard_DashboardLoading(object sender, EventArgs e) {
            SalesPersonTableAdapter categories = new SalesPersonTableAdapter();
            categories.Fill(this.nwindDataSet1.SalesPerson);
        }
    }
}
